prelude/README.md

```diff
@@ -111,6 +122,16 @@ instance categoryArr :: Category Prim.Function
 ($) :: forall a b. (a -> b) -> a -> b
 ```
 
+Applies a function to its argument
+
+```purescript
+    (+) 1 ((*) 2 (const 1 2)) = (+) 1 $ (*) 2 $ const 1 2
```

Comment:
Something using composition might make a nice example. This seems a bit dense for a beginner due to all the sections. What about something list-based like:

```purescript
length $ groupBy productCategory $ filter isInStock products
```